webapp/components/navbar.jsx

```diff
@@ -72,6 +74,23 @@ export default class Navbar extends React.Component {
         ChannelStore.addChangeListener(this.onChange);
         ChannelStore.addExtraInfoChangeListener(this.onChange);
         $('.inner-wrap').click(this.hideSidebars);
+
+        document.addEventListener('keydown', (e) => {
```

Comment:
This event should be a named function and the listener should be removed on componentWillUnmount to prevent memory leaks